sel/src/Sel/SecretKey/Stream.hs

```diff
@@ -51,6 +51,13 @@ module Sel.SecretKey.Stream
     -- ** Message Tags
   , MessageTag (..)
 
+    -- ** Additional data (AD)
+  , AdditionalData (..)
```

Comment:
Thoughts on me exposing this constructor? Or would you prefer a function like `additionalDataFromBinary` or something like that?

Reply:
If the constructor is not needed to use the API and would just be used for access, then you can just expose `additionalDataFromBinary` indeed. :)